.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+jdk:
```

Comment:
you should just be able to do:

```
git fetch upstream/master
git rebase upstream/master
git push -f
```

and the branch conflicts from the travis changes should go away.

and then do